### DISCUSSION TITLE
Movement point calculation

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#6925] [Ideas]
Currently, we have a Movement points meter in hero's info:
![image](https://user-images.githubusercontent.com/55348946/92587604-8ea3a580-f2a0-11ea-9aee-d6c725e75230.png)
It would be nice, if we add an option, that shows how much Movement Points hero would spent, when he clicks at destination point of a route.

**Comment by idshibanov** [comment]
Use `World::getDistance` to get the value.

**Comment by idshibanov** [comment]
@Branikolog where do you want this value to be displayed?

**Comment by Branikolog** [comment]
@idshibanov, sorry for late answer, missed your message in tons of PRs and new issues in my mailbox. ;)


> where do you want this value to be displayed?

I'm not sure... We can add it to the right click info window, but only when player clicks on a final destination cross ... (I don't want this information appear everywhere, but only in a cases, we have a route created) 
I'm not sure if it will look good in this window... When we click on grass, we can see "digging ok" phrase... I'm afraid, if we add there movement points counter, and it would look ugly... 
As a solution we can put a small number in a corner of this window...
Also, movement points counter can be shown, when we right click over a hero (like on screenshot in this thread) with a path created (it's the least annoying, but the most uncofortable solution).

**Comment by ihhub** [comment]
I'm reclassifying this as a discussion since no proper solution has been proposed.
